Formula/mytop.rb

```diff
@@ -32,7 +32,7 @@ class Mytop < Formula
     sha256 "d6d38a416da79de874c5f1825221f22e972ad500b6527d190cc6e9ebc45194b4"
   end
 
-  resource "DBD::mysql" do
+  resource "DBD:"mysql"" do
```

Comment:
Heh. Your script has failed you here 😉.

Reply:
there were a few others like that that I had to fix manually. just missed this one